log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -101,9 +101,6 @@ private Logging()
             ROOT.removeHandler(handler);
         }
 
-        enableConsole();
```

Comment:
instead of enabling console at the constructor level, we can wait until `configure()` step to determine which kind of console to enable

Reply:
Without digging in at all, I expect this will break something in some very subtle way.  The logging code is super prickly.  Instead, I suggest we just reconfigure the console after the initial startup

Reply:
BTW, that will likely cause issues where the console contains non-json messages, but that **might** be unavoidable if there are things logging before we are invoked.  If we really need to avoid that, then I expect we will need a system property since configuration is available much later in the startup.

Reply:
I'm not exactly sure how deep down this consoleHandler overrides. If it overrides all the way to jvm console then we'll need to keep it here. If it only overrides by the time we instantiate the logging class then I think we are fine...